pages/index.tsx

```diff
@@ -25,9 +29,9 @@ export default function HomePage() {
       title="Home"
     >
       <div className="w-full max-w-4xl mx-auto mt-32 md:mt-56">
-        <h1 className="text-xl md:text-2xl lg:text-3xl">
-          Search all R packages on CRAN and Bioconductor
-        </h1>
+        <div className="text-xl md:text-2xl lg:text-3xl">
+          Search <span className={packageCountFormat}>{packageCount}</span> R packages on CRAN and Bioconductor
```

Comment:
packageCount should be number, that will be simpler for typing, in this case you want to add some logic to have `all` and then the number if it exists 🤝